test/validations.test.js

```diff
@@ -558,7 +558,65 @@ describe('validations', function() {
   });
 
   describe('numericality', function() {
-    it('should validate numericality');
+    it('passes when given numeric values', function() {
+      User.validatesNumericalityOf('age');
+      var user = new User({age: 10});
+      user.isValid().should.be.true();
+    });
+
+    it('fails when given non-numeric values', function() {
+      User.validatesNumericalityOf('age');
+      var user = new User({age: 'notanumber'});
+      user.isValid().should.be.false();
+      user.errors.should.eql({age: ['is not a number']});
+    });
+
+    it('fails when given undefined values', function() {
+      User.validatesNumericalityOf('age');
+      var u = new User({});
+      u.isValid().should.be.false();
+      u.errors.should.eql({age: ['is blank']});
```

Comment:
@Amir-61 This user is still u, we should make them all `u` or all `user` for consistency.